std/functional.d

```diff
@@ -68,6 +68,7 @@ module std.functional;
 
 import std.meta : AliasSeq, Reverse;
 import std.traits : isCallable, Parameters;
+import std.ascii;
```

Comment:
Surely this can't be the way forward. Why exactly can't this be local?

Reply:
See the other two comments.

Reply:
I did. Still not 100% sure.